README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I think this would make most sense as a model method. This is because the controller should contain only the CRUD methods if possible and a helper method would be available across all controllers which doesn't make sense here.
 ```
```

Comment:
In this case, a Helper method would make the most sense, because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model mehtods concern how data goes to and comes from the database.